recipes/optitype/meta.yaml

```diff
@@ -20,9 +20,9 @@ requirements:
   run:
     - python <3
     - numpy =1.15.4
```

Comment:
Do you need the strict numpy pinning here?

Reply:
A good point, I think we can relax it, I've just ran a test with numpy 1.16.6 - it works
attn @andras86 please correct if the pinning is crucial